From ff693d249292da621524e36e299789065422e675 Mon Sep 17 00:00:00 2001
From: Tiejun Chen <tiejun.chen@windriver.com>
Date: Tue, 12 May 2009 10:14:03 +0800
Subject: [PATCH] fsl_mpc8360e: Protect the u-boot partition

Mask the bootloader partition as read-only on MPC836X_MDS.

[This patch should be folded into commit ID
a9e5a8b4be55c33fc48adc6fe25f2fac01679865]

Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 arch/powerpc/boot/dts/mpc836x_mds.dts |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/mpc836x_mds.dts b/arch/powerpc/boot/dts/mpc836x_mds.dts
index 526305e..7180ae6 100644
--- a/arch/powerpc/boot/dts/mpc836x_mds.dts
+++ b/arch/powerpc/boot/dts/mpc836x_mds.dts
@@ -72,6 +72,7 @@
 			partition@0 {
 				label = "U-Boot";
 				reg = <0x0 0x100000>;
+				read-only;
 			};
 
 			partition@100000 {
-- 
1.6.0.4

